README.md

```diff
@@ -10,31 +10,31 @@
 
 ## Requirements
 
-ResponseDetective is written in **Swift 2.3** and requires **iOS 8** or **OS X 10.10**.
+ResponseDetective is written in **Swift 3** and requires **iOS 8** or **OS X 10.10**.
```

Comment:
I think sentence below would be more precise:
_requires minimum_ 
or 
_requires iOS 8+ / 10.10+_